### DISCUSSION TITLE
T1 and BOLD cannot be found

### DISCUSSION THREAD

**Discussion by ghost** [HALFpipe/HALFpipe#444] [Q&A] [answered]
Hello,
we have set up Halfpipe to run preprocessing for different data sets in BIDS format and we have renamed the subjects and files as states (HALFpipe expects you to input {subject}/scan_rest.nii.gz.) However, Halfpipe cannot find any files in our working directory. 
What could be the problem?

Thanks in advance,
Franziska

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @FranzLa,

Could you upload a screenshot of your folder structure, and a screenshot of what you entered into the HALFpipe user interface?

**Reply by ghost** [reply]
Thanks four your reply, we have already resolved this issue.

**Reply by HippocampusGirl** [reply]
That’s great to hear :-)
